docs/sources/reference/commandline/cli.md

```diff
@@ -2336,7 +2336,7 @@ Running `docker stats` on multiple containers
     $ docker stats redis1 redis2
     CONTAINER           CPU %               MEM USAGE/LIMIT     MEM %               NET I/O
     redis1              0.07%               796 KiB/64 MiB      1.21%               788 B/648 B
```

Comment:
Oh, that's unfortunate. Thanks for spotting! Can you create an issue for that? (Perhaps a pull request :smile:)